devtools/startup/DevToolsStartup.jsm

```diff
@@ -1474,20 +1474,14 @@ function saveRecordingInDB(description) {
 
```

Comment:
We'll need to remove `saveRecordingInDB` here too or else we'll be hitting this old endpoint with a bad payload.

Reply:
ah good point, I opened a PR for that here https://github.com/RecordReplay/gecko-dev/pull/203